app/helpers/sessions_helper.rb

```diff
@@ -4,19 +4,41 @@ def log_in user
     session[:user_id] = user.id
   end
 
+  # Remembers a user in a persistent session.
+  def remember user
+    user.remember
+    cookies.permanent.signed[:user_id] = user.id
+    cookies.permanent[:remember_token] = user.remember_token
+  end
+
   # Returns the current logged-in user (if any).
   def current_user
-    return unless session[:user_id]
-
-    @current_user ||= User.find_by id: session[:user_id]
+    if user_id = session[:user_id]
+      @current_user ||= User.find_by id: user_id
```

Comment:
toán tử `||=` có tác dụng gì?

Reply:
||= tức là bằng biến đấy tuyển chính nó ạ, nếu current_user nil nó sẽ lấy vế sau.